src/Sylius/Component/Registry/DependencyInjection/Compiler/AliasedServicePass.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+namespace Sylius\Component\Registry\DependencyInjection\Compiler;
```

Comment:
I guess it is debateable if there should be Symfony specific couplings in the component, but it would be a shame for this to be in the bundle as my use case here is actually using it in a Symfony CMF project. If we decide to put this in we should add the DI component to `suggests` in composer.json